### ISSUE TITLE
How to enable custom hotkeys and limits not provided by GOverlay?

### ISSUE THREAD

**Issue by quidamphx** [benjamimgois/goverlay#97]
How can I enable a custom frame limit such as 120/90/60, to be cycled with the assigned hotkey? If cycling between 3 values isnt possible, something like 90/60?

Also, the frame limiter only allows me to set F1-F4 as the hotkey, and adding something like Shift_R+F1 to the config file has no effect. F1-F4 are too easily triggered by game commands with many games and adding a modifier key to it would be ideal.

**Comment by benjamimgois** [comment]
@quidamphx what version are you using ? in the latest 1.0 release toggle keys are Shift_L+F1 ... Shift_L+F4. Also you can just click the FPS that you want and the togglekey will cycle betwen than.

![image](https://github.com/benjamimgois/goverlay/assets/104206575/466ed1ad-67aa-4466-9922-65ac88df34a6)

**Comment by quidamphx** [comment]
0.9, from the Fedora repo. I'll look into installing 1.0 when it's available; I had no idea there was a newer version. Thank you, this is helpful! At least I know the features are coming, I'll wait until I can update it as sudo dnf install goverlay doesn't have any updates yet.

**Comment by benjamimgois** [comment]
No problem, application distribution is an old Linux problem. Check if 1.0 works for you

**Comment by quidamphx** [comment]
It seems to be working well since trying v1.0.

I probably didn't take the elegant or correct route, but I just downloaded and replaced the goverlay file in /usr/libexec and it launches version 1.0 from my existing shortcuts. I tried cloning the git and building and it kept coming back with an error.

Shift_R+F12 still works to open the overlay, and Shift_L+F1 does effectively toggle through the framerate options I enabled; 60, 90, 120. This is the eaxct thing I wanted and it turned out it was already there.

Thank you very much.
